src/components/tx/ErrorMessage/styles.module.css

```diff
@@ -4,6 +4,7 @@
   padding: var(--space-2);
   margin: var(--space-2) 0;
   border-radius: 4px;
+  word-break: break-all;
```

Comment:
Instead of this, I would move the echoing of the id to the details of the error.